### DISCUSSION TITLE
Can't set MX for domain

### DISCUSSION THREAD

**Discussion by mikhin** [vercel/community#5057] [Help]
### Summary

Hi all. I want to configure MX recods to handle email but it is not working. Where could be the error and how can i fix it?

<img width="1232" alt="Screenshot 2023-12-04 at 13 47 12" src="https://github.com/vercel/community/assets/38258318/cba4dd3f-2f16-4b51-ae79-ed6a00fb1469">

### Steps to Reproduce

-

**Comment by amyegan** [comment]
Please contact your email provider for help determining the necessary DNS records 🙏

**Reply by mikhin** [reply]
thanks! I have, but they do not displayed as DNS records when I check it.
